### PR TITLE
C9192 Loans: Actions while declared lost (vega) (TaaS)

### PR DESCRIPTION
**Pull request by ernestyedigaryan1**
[C9192](https://foliotest.testrail.io/index.php?/cases/view/9192)

[Allure Report](https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/5622/allure)

Screenshot 2023-11-23 at 6 33 02 PM

### CODE REVIEW

File: cypress/e2e/loans/loans-declared-lost.cy.js [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          shouldCheckIn: true,
+        });
+      });
+      Locations.deleteViaApi(testData.defaultLocation);
+      Users.deleteViaApi(testData.user.userId);
+    });
+
+    it(
+      'C9192 Loans: Actions while declared lost (vega) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.vega] },
```
Comment (dnovikau) [inline cypress/e2e/loans/loans-declared-lost.cy.js:108]: Please update with string literals

File: cypress/support/fragments/loans/changeDueDateForm.js [hunk 2]
```diff
@@ -1,5 +1,13 @@
-import { HTML } from '@interactors/html';
-import { Button, TextField, Modal, MultiColumnListCell } from '../../../../interactors';
+import { HTML, including } from '@interactors/html';
+import { matching } from 'bigtest';
```
Comment (dnovikau) [inline cypress/support/fragments/loans/changeDueDateForm.js:2]: I suppose it should be imported from '../../../../interactors';

**Review by dnovikau — APPROVED** [review]

File: cypress/e2e/loans/loans-declared-lost.cy.js (hunk 1 above)
Comment (ernestyedigaryan1) [reply cypress/e2e/loans/loans-declared-lost.cy.js]: Done

File: cypress/support/fragments/loans/changeDueDateForm.js (hunk 2 above)
Comment (ernestyedigaryan1) [reply cypress/support/fragments/loans/changeDueDateForm.js]: Done

**Review by re-jas — APPROVED** [review]

**Pull request merged**